config/routes.rb

```diff
@@ -9,7 +9,13 @@
   scope module: "staff" do
     resource :dashboard, only: :show
     resources :users
-    resources :organisations
+    resources :organisations do
+      resources :funds, only: [:new, :create]
+    end
+    resources :funds, only: [:index, :show] do
```

Comment:
From an initial click through and another visit to our wireframes I think we need to make a tweak here.

Since `fund#show` is not nested within an organisation the navigating back is jarringsince we go back to `fund#index` where we have to pick an organisation again before doing anything else. When in fact we came from `org/:id/fund/new` so we'd expect to go back to `org/:id`, I think.

I think the following routes would help with this:

```
resources :organisations do
  resources :funds, only: [:index, :show, :new, :create]
end

resources :funds do
  resources :activities, only: [:new, :create, :show]
end
```

This removes the route level `resources :activities` for showing the records, and indexing (which I don't think was used). I think we will be able to add different ways to route to activities later when we get to reporting. This is my bad in being a bit premature on the wireframe.

Have a click through with and without these changes and see if it makes sense to you. I'll attach a WIP commit as discussed as it will probably be more clear.